packages/DropdownMenu/src/Item/Item.js

```diff
@@ -0,0 +1,87 @@
+import React from "react";
+import PropTypes from "prop-types";
+import RawButton from "@paprika/raw-button";
+import ItemStyles from "./Item.styles";
+
+const { bool, func, node, string } = PropTypes;
+
+const propTypes = {
+  /** Optional custom classes */
+  className: string,
+
+  /** HTML for each item */
+  children: node.isRequired,
+
+  /** If the item is destructive. */
+  isDestructive: bool,
+
+  /**  If true, will render an <a> tag, otherwise a <span> is generated */
+  isLink: bool,
```

Comment:
Instead of creating a separate <Dropdown.Link> as in the acl-ui 3 version, I've elected to pass an `isLink` prop here which renders out the correct element. As the css styles being applied to the elements would be the same for either element i felt this should be sufficient

Reply:
Let's use `<Dropdown.Link />` since is similar to the other apis that we already have in paprika, We are trying to remove props as far we can from the components so are more easy to understand. 

```js
<DropDownMenu>
  <DropDown.Item>Some 1</DropDown.Item>
  <DropDown.Item>Some 2</DropDown.Item>
  <DropDown.Link>Link</DropDown.Link>
</DropDownMenu>
```

Reply:
This doesn't seem to be working quite right in the Storybook example currently.  The Link does get rendered as an `<a>`, but the `onClick` doesn't make sense, it needs an `href` attribute.  And likely a way to launch in a new tab (maybe the introduction of an `isExternal` prop for the `<Dropdown.Link>` would be a good idea?).